mllib/src/main/scala/org/apache/spark/ml/util/ReadWrite.scala

```diff
@@ -135,13 +135,13 @@ abstract class MLWriter extends BaseReadWrite with Logging {
 }
 
 /**
- * Trait for classes that provide [[MLWriter]].
+ * Trait for classes that provide `MLWriter`.
```

Comment:
Hmm. That's weird. At least, it should warn because I copied and pasted each class names from the messages and went to the line number. Let me double check and be back. Maybe, I made mistakes for few.